packages/react/src/components/scrollable/scrollable.decorator.tsx

```diff
@@ -1,8 +1,8 @@
-import { Addon_DecoratorFunction } from "@storybook/types";
 import React from "react";
+import { ScrollableDecorator } from "dso-toolkit";
 
-export const decorator: Addon_DecoratorFunction<JSX.Element> = (story) => (
+export const decorator: ScrollableDecorator<JSX.Element> = (story) => (
   <div id="scrollable-mock" style={{ backgroundColor: "#efefef", height: "750px", width: "500px" }}>
-    {story()}
+    {story() as React.ReactNode}
```

Comment:
Met dank aan AI Assistent van JetBrains 🫣

Reply:
Tsja, met `as` begeven we ons op een glad ijs. We zeggen eigenlijk tegen TypeScript dat wij (de devs) het beter weten.

En in mijn ervaring is dat ZELDEN het geval.

Wat denk je van:

```ts
export const decorator: ScrollableDecorator<React.JSX.Element> = (storyFn) => {
  const story = storyFn();
  
  return (
    <div id="scrollable-mock" style={{ backgroundColor: "#efefef", height: "750px", width: "500px" }}>
      {React.isValidElement(story) ? story : null}
    </div>
  );
};
```

- Kennelijk is de namespace `JSX` deprecated: `JSX.Element` omgelegd naar `React.JSX.Element`
- parameter `story` hernoemd naar `storyFn`
- return value van `storyFn()` testen met `React.isValidElement()` alvorens te renderen.

Reply:
👏